README.Rmd

```diff
@@ -28,11 +28,27 @@ Code health is reported at the level of the R package, and at the level of the T
 ### PACTA for Banks
 
 ``` {r echo = FALSE, results = "asis"}
-banks_repos <- c(
-  "RMI-PACTA/r2dii.data",
-  "RMI-PACTA/r2dii.match",
-  "RMI-PACTA/r2dii.analysis",
-  "RMI-PACTA/r2dii.plot"
+banks_repos <- list(
+  c(
+    path = "RMI-PACTA/r2dii.data",
+    lifecycle = "stable",
+    ci_check = "R.yml"
+  ),
+  c(
+    path = "RMI-PACTA/r2dii.match",
+    lifecycle = "stable",
+    ci_check = "R.yml"
+  ),
+  c(
+    path = "RMI-PACTA/r2dii.analysis",
+    lifecycle = "stable",
+    ci_check = "R.yml"
+  ),
+  c(
+    path = "RMI-PACTA/r2dii.plot",
+    lifecycle = "experimental",
+    ci_check = "R.yml"
+  )
```

Comment:
⚠️ Breaking change: I updated the expected input format of datasets to the `generate_package_table` function.

Reply:
nit: doesn't `table_lifecycle()` aim to scrape the lifecycle status from the repo's README, in which case (assuming it works reliably) in wouldn't be necessary to maintain the lifecycle here?